examples/react/css/src/hits-per-page.css

```diff
@@ -0,0 +1,30 @@
+.ais-HitsPerPage {
+  &:after {
+    transition: transform var(--ais-transition-duration)
+      var(--ais-transition-timing-function);
+    background-image: var(--ais-chevron-icon);
+    background-repeat: no-repeat;
+    content: '';
+    display: inline-block;
+    height: 1rem;
+    margin-left: calc(calc(var(--ais-spacing) * -1) + -1rem);
+    transform: translateY(0.2rem);
+    user-select: none;
+    width: 1rem;
+  }
+  .ais-HitsPerPage-select {
+    appearance: none;
+    background: none;
+    border: 0;
```

Comment:
Should we reset it here or in the `reset.css` file? Do we have other `select` in IS?